jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/form/FormProviderServiceImpl.java

```diff
@@ -70,9 +74,11 @@ public void setProviders(Set<FormProvider> providers) {
 		this.providers = providers;
 	}
 
-	private Set<FormProvider> providers;
-
+    public void setIdentityProvider(IdentityProvider identityProvider) {
```

Comment:
this will most likely have to be set from kie server as well https://github.com/droolsjbpm/droolsjbpm-integration/tree/master/kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui.

You can run any form related tests and see if they are not failing with NPE: https://github.com/droolsjbpm/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java